{"uid":"f58348c7e8e9395f","name":"Demo failed test cases","fullName":"Search Functionality: Demo failed test cases","historyId":"ce7f5c9b51f1a8e229623049aeb7cdce","time":{"start":1727287106380,"stop":1727287107670,"duration":1290},"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"C:\\Python-Selenium\\Python_SeleniumBDD\\Python_Selenium\\venv1\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python-Selenium\\Python_SeleniumBDD\\Python_Selenium\\venv1\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\testing\\steps\\login_feature.py\", line 40, in step_impl\n    assert False\n           ^^^^^\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"C:\\Python-Selenium\\Python_SeleniumBDD\\Python_Selenium\\venv1\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python-Selenium\\Python_SeleniumBDD\\Python_Selenium\\venv1\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\testing\\steps\\login_feature.py\", line 40, in step_impl\n    assert False\n           ^^^^^\n","steps":[{"name":"Given : Navigate to Home page","time":{"start":1727287106380,"stop":1727287107428,"duration":1048},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":false,"attachmentsCount":0,"shouldDisplayMessage":false},{"name":"Then Fail the test case","time":{"start":1727287107428,"stop":1727287107431,"duration":3},"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"C:\\Python-Selenium\\Python_SeleniumBDD\\Python_Selenium\\venv1\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python-Selenium\\Python_SeleniumBDD\\Python_Selenium\\venv1\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\testing\\steps\\login_feature.py\", line 40, in step_impl\n    assert False\n           ^^^^^\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":true,"attachmentsCount":0,"shouldDisplayMessage":true}],"attachments":[{"uid":"b0e7de26d5a61643","name":"failure_screenshot","source":"b0e7de26d5a61643.png","type":"image/png","size":49946}],"parameters":[],"stepsCount":2,"attachmentStep":false,"hasContent":true,"attachmentsCount":1,"shouldDisplayMessage":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Search Functionality"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"f58348c7e8e9395f.json","parameterValues":[]}